documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -64,12 +64,16 @@ $ {oc} get pods -n {namespace}
 +
 .Example output
 ----
-NAME                                  READY  STATUS   RESTARTS  AGE
-forklift-controller-788bdb4c69-mw268  2/2    Running  0         2m
-forklift-operator-6bf45b8d8-qps9v     1/1    Running  0         5m
-forklift-ui-7cdf96d8f6-xnw5n          1/1    Running  0         2m
+NAME                                                    READY   STATUS    RESTARTS   AGE
+forklift-api-bb45b8db4-cpzlg                            1/1     Running   0          6m34s
+forklift-controller-7649db6845-zd25p                    2/2     Running   0          6m38s
+forklift-must-gather-api-78fb4bcdf6-h2r4m               1/1     Running   0          6m28s
+forklift-operator-59c87cfbdc-pmkfc                      1/1     Running   0          28m
+forklift-ui-plugin-5c5564f6d6-zpd85                     1/1     Running   0          6m24s
+forklift-validation-7d84c74c6f-fj9xg                    1/1     Running   0          6m30s
+forklift-volume-populator-controller-85d5cb64b6-mrlmc   1/1     Running   0          6m36s
 ----
 +
-. If you added VMware source providers to {project-short} 2.2, upgrading to {project-short} 2.3 changes the state of any VMware providers to Critical. To fix this problem, edit the VMwhere provider by adding a VDDK `init` image and verifying the certificate of the VMware provider. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider].
+. If you added VMware source providers to {project-short} 2.2, upgrading to {project-short} 2.3 changes the state of any VMware providers to Critical. To fix this problem, edit the VMware provider by adding a VDDK `init` image. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.4/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider].
```

Comment:
the `upgrading to {project-short} 2.3` part seems awkward since users that read the 2.4 documentation are not upgrading the 2.3. it's supposed to be the same when upgrading to 2.4 but instead of changing 2.3 to 2.4, I'd drop the version we upgrade to altogether - it should be obvious that we upgrade to the latest version
```suggestion
. If you upgrade from {project-short} 2.2 and have VMware source providers defined, the state of any VMware providers would change to Critical. To fix this problem, edit the VMware provider by adding a VDDK `init` image. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.4/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider].
```